internal/eventstore/repository/gen.go

```diff
@@ -0,0 +1,4 @@
+package repository
+
+//go:generate stringer -type Operation -trimprefix Operation
+//go:generate stringer -type Field -trimprefix Field
```

Comment:
i don't get why we have to introduce stringer here

Reply:
they are used here https://github.com/zitadel/zitadel/pull/5523/files#diff-7a16771851dc78e615fdb31bc1ac789b3cc9f24859554d8fd124871f0b144b65R23

Added a comment.

Reply:
I removed the stringer implementations as I'm not sure if it changes something in an incompatible way. Didn't consider that before.